Archiver.py

```diff
@@ -28,22 +54,66 @@ def unzip_string(processed_string):
     processed_list = list(map(int, processed_string.split()))
     additional_sequence = {}
     number_additional_sequence = 256
-    processed_sequense = chr(processed_list[0])
-    uncompressed_string = processed_sequense
+    processed_sequence = chr(processed_list[0])
+    uncompressed_string = processed_sequence
     for counter_list in range(1, len(processed_list)):
         if processed_list[counter_list] < 256:
             uncompressed_sequence = chr(processed_list[counter_list])
         else:
             uncompressed_sequence = additional_sequence[processed_list[counter_list]]
         uncompressed_string += uncompressed_sequence
-        processed_sequense += uncompressed_sequence[0]
-        additional_sequence[number_additional_sequence] = processed_sequense
+        processed_sequence += uncompressed_sequence[0]
+        additional_sequence[number_additional_sequence] = processed_sequence
         number_additional_sequence += 1
-        processed_sequense = uncompressed_sequence
+        processed_sequence = uncompressed_sequence
     return uncompressed_string
 
 
+def menu(command_message="begin", file_path=None):
+    #  Output menu
+    if command_message == "begin":
+        with open("menu.txt") as menu_file:
+            menu_text = "".join(menu_file.readlines())
+        menu_main = "Select option(print number without point):"
+        if file_path:
+            menu_status = "Selected file " + os.path.basename(file_path)
+        else:
+            menu_status = "No one file is selected"
+        print(f"\n{menu_main} \n{menu_status} \n{menu_text}")
+        new_command_message = str(input())
+        menu(new_command_message, file_path)
+    #  1. Choose file
+    elif command_message == "1":
+        input_file_path = str(input("Print file path or file name: "))
+        input_text, input_file_name = read_file(input_file_path)
+        if input_text:
+            print("File chosen successfully")
+            menu("begin", input_file_path)
+        else:
+            menu()
+    #  2. Compress chosen file
+    elif command_message == "2":
+        input_text, input_file_name = read_file(file_path)
```

Comment:
General note: returning a set is probably not the best idea. I would recommend to return dictionary from read_file() method